centreon/src/App/ActivityLogging/TestCommand.php

```
@@ -0,0 +1,63 @@
+)]
+class TestCommand extends Command
+{
+    public function __construct(
+
+        #[Autowire(service: 'doctrine.dbal.default_connection')]
+        private Connection $connection
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

PHPMD (2.15.0)

Avoid unused parameters such as '$input'. (undefined)

(UnusedFormalParameter)